### ISSUE TITLE
[Phase 3] TTS data download, preprocessing & eval set

### ISSUE THREAD

**Issue by selcuk-yalcin** [selcuk-yalcin/Kurdish-LLM-FineTuning#7]
## Summary

Download selected Kurmanji speech datasets, normalize audio, align transcripts, and create train/val/test splits plus a held-out TTS eval set.

## Prerequisites

- [ ] [#6](https://github.com/selcuk-yalcin/Kurdish-LLM-FineTuning/issues/6) TTS datasets selected and enabled in config

## Tasks

- [ ] Install TTS deps: `pip install -r requirements-tts.txt`
- [ ] Install system dep: `brew install ffmpeg` (macOS)
- [ ] Run `python scripts/tts/download_data.py`
- [ ] Run `python scripts/tts/preprocess.py`:
  - Normalize: mono, target sample rate (22–24 kHz)
  - Trim silence, validate duration
  - Latin Kurmanji transcript filter
  - Train / val / test splits
- [ ] Build TTS eval set: **20+ held-out sentences** in `data/tts/eval/`
- [ ] Run data stats report (hours, speakers, avg duration)
- [ ] @cem-acun: listen to 10 random samples — note quality issues

## Data format (target)

```json
{
  "id": "tts-001",
  "text": "Silav, çawa yî?",
  "audio_path": "data/tts/raw/audio/001.wav",
  "sample_rate": 22050,
  "duration_sec": 2.4,
  "speaker_id": "speaker_01"
}
```

## Acceptance criteria

- Processed TTS data ready under `data/tts/processed/`
- Eval set with 20+ pairs
- Stats logged; quality acceptable for fine-tuning

## Parallel work

**Independent from translation track** once [#6](https://github.com/selcuk-yalcin/Kurdish-LLM-FineTuning/issues/6) is done.

**Can overlap with:** @selcuk-yalcin running [#8](https://github.com/selcuk-yalcin/Kurdish-LLM-FineTuning/issues/8) XTTS zero-shot baseline on same eval text.

## Resources

- [scripts/tts/download_data.py](https://github.com/selcuk-yalcin/Kurdish-LLM-FineTuning/blob/main/scripts/tts/download_data.py)
- [scripts/tts/preprocess.py](https://github.com/selcuk-yalcin/Kurdish-LLM-FineTuning/blob/main/scripts/tts/preprocess.py)
- [src/tts/preprocessor.py](https://github.com/selcuk-yalcin/Kurdish-LLM-FineTuning/blob/main/src/tts/preprocessor.py)
- [config/tts/datasets.yaml](https://github.com/selcuk-yalcin/Kurdish-LLM-FineTuning/blob/main/config/tts/datasets.yaml)

## Assignee

@cem-acun — download, preprocess, audio quality check

**Comment by selcuk-yalcin** [comment]
Consolidated into streamlined 35-issue roadmap (Aug 2026).
